storage/rocksdb/rdb_datadic.cc

```diff
@@ -780,6 +783,88 @@ int Rdb_key_def::successor(uchar *const packed_tuple, const uint &len) {
   return changed;
 }
 
+void Rdb_key_def::get_lookup_bitmap(TABLE *const table, MY_BITMAP *map) const {
+  bitmap_init(map, nullptr, 16, false);
+  uint maybe_covered_columns = 0;
+  uint curr_bitmap_pos = 0;
+
+  for (uint i = 0; i < m_key_parts; i++) {
```

Comment:
I think there may be problems in your accounting if you have duplicated columns in your key since maybe_covered_columns will be incremented twice, while requested_columns is derived from read_set.